modules/SocketManager/module/inc/SocketManager/socketmanager_config.h

```diff
@@ -158,6 +158,16 @@
 #define SOCKETMANAGER_CONFIG_MAX_SOCKETS 1024
 #endif
 
+/**
+ * SOCKETMANAGER_CONFIG_MAX_POLL_TIMEOUT
+ *
+ * Restrict the poll time out (in ms) to improve responsiveness. -1 for infinite */
+
+
+#ifndef SOCKETMANAGER_CONFIG_MAX_POLL_TIMEOUT
+#define SOCKETMANAGER_CONFIG_MAX_POLL_TIMEOUT 1024
+#endif
+
```

Comment:
We ought to get feedback from @kenchiang here ... i though he wanted this hidden behind compile time options, to prevent the new code from getting used if its not specifically requested ...   i would think this defined the value in all cases...

Reply:
Actually, this code is automatically generated from SocketManager.yml.

Reply:
This is an auto generated file based on SocketManager.yml